### PR TITLE
Add brief historical perspective in acknowledgements.

### PR DESCRIPTION
**Pull request by msporny**
This adds a brief historical perspective on the specification before it entered the DID WG and acknowledges/appreciates the individuals that worked on the proto-documents that led to the DID Specification.

See the new text here: https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/790.html#acknowledgements

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/790.html" title="Last updated on Sep 11, 2021, 10:21 PM UTC (77dab87)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/790/c672e55...77dab87.html" title="Last updated on Sep 11, 2021, 10:21 PM UTC (77dab87)">Diff</a>

### CODE REVIEW

**Review by ChristopherA — APPROVED** [review]
+1

**Review by kdenhartog — APPROVED** [review]

**Comment by iherman** [comment]
Before merging *the echidna action should be switched off* (I guess the yml file should be removed) to avoid the merge leading to a new TR publication.

**Comment by msporny** [comment]
> Before merging _the echidna action should be switched off_ (I guess the yml file should be removed) to avoid the merge leading to a new TR publication.

I have disabled Echidna auto-publish here: https://github.com/w3c/did-core/actions/workflows/auto-publish.yml

**Review by talltree — APPROVED** [review]
Manu, good job summarizing a lot of history in a short space.

File: index.html
Comment (TallTed) [inline index.html:6356]:
```suggestion
Group. In particular, this includes the work of Phil Zimmerman, Jon Callas,
Lutz Donnerhacke, Hal Finney, David Shaw, and Rodney Thayer on <a
```

**Review by TallTed — CHANGES_REQUESTED** [review]
One tweak, to broaden the influential "tech and specs" which I think includes more than PGP, though that may be all that gets called out here.

File: index.html
Comment (peacekeeper) [inline index.html:6367]: Suggestion to also mention XDI.org. My recollection is that members of the W3C Web Payments Community Group and members of the XDI.org Registry Working Group came together to do the initial work at RWoT.

```suggestion
Community Group's work led by Dave Longley and Manu Sporny.
Around the same time, the XDI.org Registry Working Group <a href="https://docs.google.com/document/d/1EP-KhH60y-nl4xkEzoeSf3DjmjLomfboF4p2umF51FA/">began exploring</a> decentralized
technologies for replacing its existing identifier registry.
Some of the <a
```

File: index.html
Comment (peacekeeper) [inline index.html:6369]: Suggestion to also link to the very early and important DPKI paper. Also, this way we'd have links to papers from both RWoT1 and RWoT2.

```suggestion
Some of the first <a
href="https://github.com/WebOfTrustInfo/rwot1-sf/blob/master/final-documents/dpki.pdf">written</a> <a
href="https://github.com/WebOfTrustInfo/rwot2-id2020/blob/master/final-documents/requirements-for-dids.pdf">papers</a> exploring the concept of Decentralized Identifiers can be
```

**Review by peacekeeper — COMMENTED** [review]
Looks good, here are 2 small suggestions.

File: index.html
Comment (dlongley) [reply index.html:6367]: JFYI, the XDI.org work happened (I believe) ~1.5 years later, so we should not say "around the same time".

Comment (peacekeeper) [reply index.html:6367]: I think the XDI.org work started in mid 2015, yes that's later than the Web Payments work, but if the temporal context is "the mid-2010s", then I'd say it's still "around the same time". But we can also change it to "soon after" or something similar, do you have a suggestion?

Comment (dlongley) [reply index.html:6367]: "Around the same time" => "Around a year later" sounds like it would work.

**Review by OR13 — APPROVED** [review]

File: index.html
Comment (OR13) [reply index.html:6367]: is `https://docs.google.com/document/d/1EP-KhH60y-nl4xkEzoeSf3DjmjLomfboF4p2umF51FA/` the best link?

Comment (peacekeeper) [reply index.html:6367]: > "Around the same time" => "Around a year later" sounds like it would work.

Changed that.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-08-17-did#section3) on 2021-08-17

List of resolutions:

- ***[Resolution No. 1](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-08-17-did#resolution1):  The WG will add editorial text to the acknowledgements section of the Proposed Recommendation that gives some historical context for the work and thanks those who were involved.***

<details><summary><i>View the transcript</i></summary>

### 3. historical PR 

_See github pull request [#790](https://github.com/w3c/did-core/pull/790)._




**Brent Zundel:** Anything else on this topic?  
… Manu has written a PR about the historical context of the DID spec  
… It's to acknowledge past work, people, and efforts.  
… Please review if you want to tweak this and add suggestions  

**Manu Sporny:** It's currently written as the editors writing this, I'd like to upgrade it to the WG writing it.  

> *Joe Andrieu:* +1 for speaking "from the WG"

**Manu Sporny:** I don't think there is anything controversial, we'd like to acknowledge the work before there was a DID WG.  

> *bumblefudge:* +1

> *Kaliya Young:* I went back and looked up past IIW Sessions that foreshadow the group - they are there in 2015 - [https://iiw.idcommons.net/index.php?title=Special%3ASearch&search=blockchain&go=Go](https://iiw.idcommons.net/index.php?title=Special%3ASearch&search=blockchain&go=Go)

**Manu Sporny:** We'd like to give people an understanding that this work started long ago.  

> *Drummond Reed:* +1 to that change.

**Manu Sporny:** Would anyone object to change this from "the editors acknowledge" to "the WG acknowledges", or similar  

> *Dave Longley:* +1 to "The Working Group acknowledges"

**Ivan Herman:** It's more appropriate if it says "the WG" rather than "the editors"  

> *Manu Sporny:* agree, Ivan -- the only reason we put "Editors" is to avoid process objections...

> *Manu Sporny:* whatever those might be

> *Ted Thibodeau Jr.:* +1 spread the love to as many of the shoulders on which we stand as we can recall

> *Charles Lehner:* +1

> *Markus Sabadello:* +1

**Ivan Herman:** This is only editorial, so I don't think anyone would object from a process point of view.  

**Manu Sporny:** Should we document a WG resolution, to make absolutely sure?  

**Brent Zundel:** Any changes to the proposal?  

> **Proposed resolution: The WG will add editorial text to the acknowledgements section of the Proposed Recommendation that gives some historical context for the work and thanks those who were involved.** *(Brent Zundel)*

> *Manu Sporny:* +1

> *Drummond Reed:* +1

> *Brent Zundel:* +1

> *Shigeya Suzuki:* +1

> *Dave Longley:* +1

> *Adrian Gropper:* +1

> *Joe Andrieu:* +1

> *Ivan Herman:* +1

> *Markus Sabadello:* +1

> *Kaliya Young:* +1

> *Orie Steele:* +1

> *Ted Thibodeau Jr.:* +1

> ***Resolution #1: The WG will add editorial text to the acknowledgements section of the Proposed Recommendation that gives some historical context for the work and thanks those who were involved.***

**Ivan Herman:** Do we say somewhere in the document that all appendices are non-normative?  
… If not, then we should mark acknowledgements as being explicitly non-normative.  

**Manu Sporny:** It doesn't say that, I can add it. We have it for all other sections.  

> *Drummond Reed:* Yes, Appendix A says non-normative explicitly.

**Manu Sporny:** I'm going to raise an issue.  

**Brent Zundel:** Anything else on this topic?  

</details>

File: index.html
Comment (peacekeeper) [reply index.html:6367]: > is `https://docs.google.com/document/d/1EP-KhH60y-nl4xkEzoeSf3DjmjLomfboF4p2umF51FA/` the best link?

Yes, this was a series of meetings over a few months where all the work was done via Google Docs..

File: index.html
Comment (rxgrant) [inline index.html:6381]: As with the other links to RWOT documents, let's use the final version rather than the rough draft.  As with the attribution theme that led the WG to consider this section, let's use the available evidence, which here is the author and contributor list at the end of the document.

```suggestion
Further refinement at the Rebooting the Web of Trust workshops led to the <a
href="https://github.com/WebOfTrustInfo/rwot3-sf/blob/master/final-documents/did-implementer-draft-10.pdf">first
implementers documentation</a>, edited by Drummond Reed, Les Chasen, Christopher Allen, 
and Ryan Grant. Contributors included Manu Sporny, Dave Longley, Jason Law, Daniel Hardman,
Markus Sabadello, Christian Lundkvist, and Jonathan Endersby. 
This initial work was then merged into the W3C Credentials
```

**Review by rxgrant — CHANGES_REQUESTED** [review]
I'm requesting these changes.

File: index.html
Comment (msporny) [reply index.html:6381]: LGTM.

**Review by talltree — APPROVED** [review]

**Review by peacekeeper — APPROVED** [review]

**Review by burnburn — APPROVED** [review]

File: index.html
Comment (TallTed) [inline index.html:6353]:
```suggestion
specification, and extend sincere appreciation to those individuals that
```

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-08-24-did#section5) on 2021-08-24

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 5. DID core issues 

_See github pull request [#792](https://github.com/w3c/did-core/pull/792), [#790](https://github.com/w3c/did-core/pull/790)._




**Manu Sporny:** Just wanted to draw attention to the two issues on DID Core.  

**Daniel Burnett:** Alrighty.  
… Since we don't have Joe on at the moment... let's just spend a few minutes on that, Manu.  

**Manu Sporny:** There are two PRs for DID Core: one of them is the acknowledgement section - I believe that is completely ready to merge... Well, there's one minor thing and then it's ready to merge.  
… I'm not hitting the merge button until the votes are in. I think that's the right thing to do? It won't autopublish...  
… The other Pull Request is for... Charles made a representation-specific entries change to the spec. I believe it's largely editorial.  
… I suggest we don't pull that in until after the votes are in and we're cleaning up the recommendation document.  
… Two questions: is that the process the chairs want? And does the group see any issue with pulling in these PRs?  
… Charles, it's strange to make changes to the specification after putting it to vote. But I think they're totally okay as it's editorial.  

> *Markus Sabadello:* +1 they're editorial

**Manu Sporny:** "mostly editorial" -> I am asserting both of these things are entirely editorial - and would be surprised if anyone said otherwise.  

**Drummond Reed:** I've looked at them and agree it's editorial. Not an issue.  

> *Drummond Reed:* I also agree not to pull them into the final until after the vote is done.

**Daniel Burnett:** I'm not hearing any objections to pulling them in - when the time is right.  
… I guess we could do it in the working copy... but they wouldn't apply until we get to the final specification.  

**Manu Sporny:** Editors... are we good to go?  

> *Ted Thibodeau Jr.:* +1 merge when it feels right

**Brent Zundel:** To my understanding, he is satisfied.  

> *Drummond Reed:* +1

</details>

File: index.html
Comment (msporny) [inline index.html:6352]:
```suggestion
The Working Group gratefully acknowledges the work that led to the creation of this
```

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no sustained objections, WG resolution to merge, merging.

File: index.html
Comment (TallTed) [inline index.html:6344]: Noticed it after you merged it...
```suggestion
The Working Group extends deep appreciation and heartfelt thanks to our Chairs,
```

**Pull request merged**
